### DISCUSSION TITLE
Welcome to helios-mirror Discussions!

### DISCUSSION THREAD

**Discussion by arshsisodiya** [arshsisodiya/helios-mirror#72] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by star-lord-29** [comment]
Heyy! i am a 20yo college graduate. Im on holidays now, so thought I'd do something new, but strangely ended up here. and I have a question - is GDToT links supported to mirror automatically? or do I have to edit in Direct Link Generator?

**Reply by arshsisodiya** [reply]
> Heyy! i am a 20yo college graduate. Im on holidays now, so thought I'd do something new, but strangely ended up here. and I have a question - is GDToT links supported to mirror automatically? or do I have to edit in Direct Link Generator?

I'm Replying this after 26 days but yes Gdtot and Appdrive links can be cloned directly without any edit in code

**Comment by ghost** [comment]
I'm deployed ur Helios mirror leech bot but when I use /search cmd and search torrent it gives me a html file but i want it in telegraph page. Give any solution

**Reply by arshsisodiya** [reply]
That html file thing is a feature, Telegraph have some page limits so we can't output all results in Telegraph that's why we are using html for Torrent search

**Comment by ghost** [comment]
Thanks i thought I made a mistake while deploying the bot.
